src/middlewared/middlewared/plugins/failover_/reboot.py

```diff
@@ -0,0 +1,71 @@
+import asyncio
+import errno
+import time
+
+from middlewared.schema import accepts, returns
+from middlewared.service import CallError, job, Service
+
+
+class FailoverService(Service):
+
+    class Config:
+        cli_private = True
+        role_prefix = 'FAILOVER'
+
+    @accepts()
+    @returns()
+    @job(lock='reboot_standby')
+    async def reboot_standby(self, job):
+        """
+        Reboot the standby node and wait for it to come back online.
+        """
+        remote_boot_id = await self.middleware.call('failover.call_remote', 'system.boot_id')
+
+        job.set_progress(5, 'Rebooting standby controller')
+        await self.middleware.call(
+            'failover.call_remote', 'system.reboot',
```

Comment:
Okay, we need to change this to `failover.call_remote failover.become_passive` which will forcefully and immediately reboot the other node. Then we can remove the big long-winded comment right below here that starts with `SCALE is using system and at the time of writing...`